bigbluebutton-html5/imports/api/2.0/users/server/eventHandlers.js

```diff
@@ -7,6 +7,7 @@ import handleEmojiStatus from './handlers/emojiStatus';
 import handleGetUsers from './handlers/getUsers';
 import handleGuestsWaitingForApproval from './handlers/guestsWaitingForApproval';
 import handleGuestApproved from './handlers/guestApproved';
+import handleUserKicked from './handlers/userKicked';
```

Comment:
It's probably better to follow the event name and swap `kicked` with `ejected` everywhere, including Users collection and client-side checks.

Reply:
I don't think we can _fully_unify the action since on the client side it's called `kicked` and on akka-apps it's `ejected`. I'd pick `kick` over `eject` if we were to fully unify it.

Reply:
We can have both, can't we?
The action under 'users/server/methods' can be "kickUser", but event handler, modifier and a property in the collection - 'eject', since they are triggered by a 'UserEjectedFromMeetingEvtMsg'.
This way we won't have to make any exceptions in our naming.